src/compiler/path.ts

```diff
@@ -452,7 +452,9 @@ namespace ts {
      * Normalize path separators, converting `\` into `/`.
      */
     export function normalizeSlashes(path: string): string {

```

Comment:
Funny question, should this entire function be a no-op on *nix? .e.g. conditionally define it based on the platform?

Reply:
Strictly speaking, the user can enter their own slashes.

Reply:
One of these days I'll find someone who is actually using `\\` on *nix in the manner it is allowed to be used, i.e. as part of the name of a file or directory, then watch all hell break loose from the tools that are designed to convert them to slashes.

Reply:
That's hilarious and we're definitely doing it wrong.  I actually just meant that you could use `\\` in paths in tsconfig.json.

Reply:
In all honesty, we should probably make the presence of `\\` in tsconfig file paths (or source code paths) an error, since it means that the paths aren't strictly portable. Have been considering the same thing for the config files in Rush.

Reply:
Filed #44174 /cc @DanielRosenwasser